specification/gedcom-3-structures-3-meaning.md

```diff
@@ -960,7 +960,15 @@ See also `INDIVIDUAL_ATTRIBUTE_STRUCTURE`.
 
 #### `NICK` (Nickname) `g7:NICK`
 
-A descriptive or familiar name that is used instead of, or in addition to, one’s proper name.
+A descriptive or familiar name that is used instead of, or in addition to, one’s official or legal name.
+
+:::note
+The title and description of this structure was introduced with version 5.5 in 1996, but are understood differently by different users.
+Some use NICK only for names that would be inappropriate in formal settings.
+Some use it for pseudonyms regardless of where they are used.
+Some use it for any variant of a name that is not the one used on legal documents 
+Because all of these uses, and likely others as well, are common in existing data, no further clarification of the meaning of the NICK structure is possible without contradicting some existing data.
+:::
```

Comment:
```suggestion
:::note
The label "nickname" and description text of this structure were introduced with version 5.5 in 1996, but are understood differently by different users.
Some use `NICK` only for names that would be inappropriate in formal settings.
Some use it for pseudonyms regardless of where they are used.
Some use it for any variant of a name that is not the one used on legal documents.
Because all of these uses, and likely others as well, are common in existing data, no further clarification of the meaning of the `NICK` structure is possible without contradicting some existing data.
:::
```